dashboard/app/views/levels/_external.html.haml

```diff
@@ -1,9 +1,15 @@
 %link{rel: 'stylesheet', href: asset_path('blockly/css/common.css')}
+- href = asset_path @level.properties['href']
+- is_markdown = href.end_with? '.md'
+= javascript_include_tag 'Showdown.min.js' if is_markdown
 
 #external
   :javascript
     var options = #{@level.properties['options'].to_json}
-
-    $.ajax("#{asset_path @level.properties['href']}").done(function (data) {
+    var href = "#{href}"
+    $.ajax(href).done(function (data) {
+      if(#{is_markdown}) {
+        data = new Showdown.converter().makeHtml(data);
```

Comment:
We already have the 'redcarpet' gem in our Gemfile to convert Markdown -> HTML.  There's a helper so you can do `TextRender.file 'example.md'`, see: https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/pegasus/text_render.rb.

My thought is that it's better to use the exiting gem rather than add a new dependency, especially when the rendered HTML will get cached by Varnish.

It also makes `_external.html.haml` cleaner since that view doesn't need to know about Markdown vs. HTML, it can just ask for the resource and Rails will take care of the conversion.